### PR TITLE
Added oembed in docs

### PR DESCRIPTION
**Pull request by sonakshs**
image

### CODE REVIEW

File: docs/jvn/oembed.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## Oembed
```
Comment (PrajwalPrashanth) [inline docs/jvn/oembed.md]: Make this as `Embedding Notebooks to your site`. As the top heading will be title on the side bar.

File: docs/index.rst [hunk 2]
```diff
@@ -46,6 +46,7 @@ where even Students and Enthusiasts can leverage the same and also use
   jvn/logger
   jvn/notif
   jvn/cli
+  jvn/oembed
```
Comment (PrajwalPrashanth) [inline docs/index.rst]: you can move it to the getting started section just before `Jovian Pro`

File: docs/jvn/oembed.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+#### Embedding Notebooks to your site
+
+API:
```
Comment (PrajwalPrashanth) [inline docs/jvn/oembed.md]: Can you make it step by step process? to explain whats to be done.

**Comment by PrajwalPrashanth** [comment]
https://prajwalprashanth-demo.readthedocs.io/en/ss-add-oembed-docs/jvn/oembed.html

**Comment by aakashns** [comment]
This needs more explanation. It's not clear what is documented on the page. Try writing some description assuming that the user doesn't know anything about Oembed (which is try for most people), and is just interested in embedding notebooks somewhere (on their blog, website etc.)

**Pull request merged**